src/Ui.Altair/AltairExtensions.cs

```diff
@@ -0,0 +1,19 @@
+using GraphQL.Server.Ui.Altair;
+
+namespace Microsoft.AspNetCore.Builder
+{
+    /// <summary>
+    /// Extension methods for <see cref="IApplicationBuilder"/>
+    /// </summary>
+    public static class PlaygroundExtensions
```

Comment:
```suggestion
    public static class AltairExtensions
```

Reply:
Sorry for the oversight. It was a copy-paste job 😄